pkg/services/sqlstore/searchstore/filters.go

```diff
@@ -87,10 +87,16 @@ func (f TitleFilter) Where() (string, []interface{}) {
 }
 
 type FolderFilter struct {
-	IDs []int64
+	Dialect migrator.Dialect
+	IDs     []int64
 }
 
 func (f FolderFilter) Where() (string, []interface{}) {
+	if len(f.IDs) == 1 && f.IDs[0] == 0 {
```

Comment:
what happens if `f.IDs` is `[0,1]`? Is that a use-case we need to worry about?

Reply:
It might be same problem, but don't think it's a use case to worry about for the moment.